### PR TITLE
CCM-2310: Add success tests for dev for /v1/messages

### PR DESCRIPTION
**Pull request by seanmarjason-nhs**
## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Checklist
* [ ] Brief description of work completed, and any technical decisions made as part of the PR
* [ ] PR link added as a comment to the relevant JIRA ticket
* [ ] Branch deployed to a dynamic env - link to deployment pipeline
* [ ] PR link shared on Slack and/or Teams
* [ ] 1 reviews received

### CODE REVIEW

File: tests/development/messages/create_message/test_success.py [hunk 1]
```diff
@@ -0,0 +1,103 @@
+import requests
+import pytest
+import uuid
+from lib import Assertions, Generators
+from lib.constants.messages_paths import MESSAGES_ENDPOINT
+import lib.constants.constants as constants
+
+CORRELATION_ID = [None, "228aac39-542d-4803-b28e-5de9e100b9f8"]
+DUPLICATE_ROUTING_PLAN_TEMPLATE_ID = "bb454d66-033b-45c0-bbb6-d9b3420a0bd4"
```
Comment (PhilipSkinner) [inline tests/development/messages/create_message/test_success.py:9]: Probably not required

File: tests/development/messages/create_message/test_success.py [hunk 2]
```diff
@@ -0,0 +1,103 @@
+import requests
+import pytest
+import uuid
+from lib import Assertions, Generators
+from lib.constants.messages_paths import MESSAGES_ENDPOINT
+import lib.constants.constants as constants
+
+CORRELATION_ID = [None, "228aac39-542d-4803-b28e-5de9e100b9f8"]
```
Comment (PhilipSkinner) [inline tests/development/messages/create_message/test_success.py:8]: Should use the standard set of correlation ids from `constants`

File: tests/lib/assertions.py [hunk 3]
```diff
@@ -46,10 +46,11 @@ def assert_200_response_message(resp, environment):
         hostname = f"{environment}.api.service.nhs.uk"
         prefixes = ["internal-dev", "internal-qa"]
 
-        for p in prefixes:
-            if p in response.get("links").get("self"):
-                hostname = f"{p}-{hostname}"
-                break
+        if environment == 'sandbox':
```
Comment (PhilipSkinner) [inline tests/lib/assertions.py:49]: Are we sure this shouldn't be getting applied globally? What was the reason for making it sandbox only?

Comment (seanmarjason-nhs) [reply tests/lib/assertions.py:49]: Ended up with the test looking for internal-dev-dev in the links ... we don't need to be appending anything when its a non-sandbox environment, just use the environment passed directly in

Comment (PhilipSkinner) [reply tests/lib/assertions.py:49]: nice ok! just thought I'd check

**Review by PhilipSkinner — APPROVED** [review]

**Review by Kennedy1265 — APPROVED** [review]

**Pull request merged**